impl/windows/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./windows');
```

Comment:
Could we remove this file and rename `impl/windows/windows.js` to `impl/windows/index.js`?